static_report/package.json

```diff
@@ -79,11 +79,13 @@
     "build": "run-s strip:html rm:cli-reports && run-p -l build:single build:comparison",
     "build:single": "npm run bundle:norm:single && mv build/single-report ../piperider_cli/data/report/single-report",
     "build:comparison": "npm run bundle:norm:comparison && mv build/comparison-report ../piperider_cli/data/report/comparison-report",
-    "bundle:norm:single": "dotenv -e .env.single craco build",
-    "bundle:case:single-edge": "dotenv -e .env.single-edge craco build",
-    "bundle:norm:comparison": "dotenv -e .env.comparison craco build",
-    "bundle:case:comparison-asymmetric": "dotenv -e .env.comparison-asymmetric craco build",
-    "bundle:e2e:norm": "run-p -l bundle:norm:*",
+    "bundle:norm:single": "dotenv -e ./build-env/.env.single craco build",
+    "bundle:case:single-edge": "dotenv -e ./build-env/.env.single-edge craco build",
+    "bundle:norm:comparison": "dotenv -e ./build-env/.env.comparison craco build",
+    "bundle:case:comparison-asymmetric": "dotenv -e ./build-env/.env.comparison-asymmetric craco build",
+    "bundle:e2e:single": "dotenv -e ./build-env/.env.single-e2e craco build",
+    "bundle:e2e:comparison": "dotenv -e ./build-env/.env.comparison-e2e craco build",
+    "bundle:e2e": "run-p -l bundle:e2e:*",
```

Comment:
Adjustment e2e testing environment, and also moving `.env*` files to `build-env` folder.